tests/run/isolated_limited_api_tests.srctree

```diff
@@ -31,7 +31,7 @@ modules = [
     "limited_multi_phase"
 ]
 
-for version in range(8, sys.version_info[1]+1):
+for version in range(9, sys.version_info[1]+1):
```

Comment:
These tests were trying to test `Py_LIMITED_API=0x03080000` which won't work now (and was crashing), so I've pushed a commit to drop that.

That does suggest we should also add an error on a too low Limited API versions too. I'll do a new PR for that though